examples/benchmarks/bert/src/bert_layers.py

```diff
@@ -425,6 +499,7 @@ def __init__(self, config):
             (1, self.num_attention_heads, self._current_alibi_size,
              self._current_alibi_size))
         self.rebuild_alibi_tensor(size=config.alibi_starting_size)
+        self.slopes = None
```

Comment:
Hey @Skylion007 many thanks for this PR! I am currently testing it (with own dataset) and training is working (8x H100).

I had to remove this line, because:

* `this.slopes` is set in the `rebuild_alibi_tensor` function
* it is later needed in line 583

Setting to `None` will then cause an error in line 583.

Reply:
This was on me trying to appease the linting gods. Thanks for catching!